<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="home-pc" failures="2" tests="18" name="Surefire test" time="88.532" errors="0" timestamp="2022-10-14T17:06:06 MSK">
  <testcase classname="CreateNewCourierTest" name="tryCreateDuplicateLoginCourierThenConflict on tryCreateDuplicateLoginCourierThenConflict(CreateNewCourierTest)" time="9.565"/>
  <testcase classname="CreateNewCourierTest" name="tryCreateDuplicateCourierThenConflict on tryCreateDuplicateCourierThenConflict(CreateNewCourierTest)" time="10.808"/>
  <testcase classname="CreateNewCourierTest" name="createCourierWithoutFirstNamePositiveTest on createCourierWithoutFirstNamePositiveTest(CreateNewCourierTest)" time="11.793"/>
  <testcase classname="CreateNewCourierTest" name="createCourierPositiveTest on createCourierPositiveTest(CreateNewCourierTest)" time="12.673"/>
  <testcase classname="CreateNewCourierTest" name="tryCreateCourierWithoutLoginReturnBadRequest on tryCreateCourierWithoutLoginReturnBadRequest(CreateNewCourierTest)" time="12.956"/>
  <testcase classname="CreateNewCourierTest" name="tryCreateCourierWithoutPasswordReturnBadRequest on tryCreateCourierWithoutPasswordReturnBadRequest(CreateNewCourierTest)" time="13.263"/>
  <testcase classname="LoginCourierTest" name="tryAuthCourierWithoutLoginReturnBadRequest on tryAuthCourierWithoutLoginReturnBadRequest(LoginCourierTest)" time="0.292"/>
  <testcase classname="LoginCourierTest" name="tryAuthCourierIncorrectLoginThenNotFound on tryAuthCourierIncorrectLoginThenNotFound(LoginCourierTest)" time="1.418"/>
  <testcase classname="LoginCourierTest" name="tryAuthCourierWithoutPasswordReturnBadRequest on tryAuthCourierWithoutPasswordReturnBadRequest(LoginCourierTest)" time="61.784">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;400&amp;gt; but was &amp;lt;504&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <504>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at ru.yandex.practikum.utils.Utils.checkResponseReturnBadRequestCode(Utils.java:21)
at LoginCourierTest.tryAuthCourierWithoutPasswordReturnBadRequest(LoginCourierTest.java:44)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- tryAuthCourierWithoutPasswordReturnBadRequest on tryAuthCourierWithoutPasswordReturnBadRequest(LoginCourierTest) -->
  <testcase classname="LoginCourierTest" name="authCourierPositiveTest on authCourierPositiveTest(LoginCourierTest)" time="63.114"/>
  <testcase classname="LoginCourierTest" name="tryAuthNonexistentCourierThenNotFound on tryAuthNonexistentCourierThenNotFound(LoginCourierTest)" time="64.316"/>
  <testcase classname="LoginCourierTest" name="tryAuthCourierIncorrectPasswordThenNotFound on tryAuthCourierIncorrectPasswordThenNotFound(LoginCourierTest)" time="65.471"/>
  <testcase classname="OrderListTest" name="getOrderListForCreatedCourierNotNull on getOrderListForCreatedCourierNotNull(OrderListTest)" time="3.136">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;200&amp;gt; but was &amp;lt;404&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at ru.yandex.practikum.utils.Utils.checkResponseReturnSuccessCode(Utils.java:13)
at OrderListTest.getOrdersCount(OrderListTest.java:87)
at OrderListTest.getOrderListForCreatedCourierNotNull(OrderListTest.java:50)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 41 stack frames]]>
    </failure>
  </testcase> <!-- getOrderListForCreatedCourierNotNull on getOrderListForCreatedCourierNotNull(OrderListTest) -->
  <testcase classname="OrderListTest" name="getTotalOrderListNotNull on getTotalOrderListNotNull(OrderListTest)" time="6.894"/>
  <testcase classname="OrderParamTest" name="placeOrderDifferentColorsPositiveTest[0] on placeOrderDifferentColorsPositiveTest[0](OrderParamTest)" time="0.645"/>
  <testcase classname="OrderParamTest" name="placeOrderDifferentColorsPositiveTest[1] on placeOrderDifferentColorsPositiveTest[1](OrderParamTest)" time="1.266"/>
  <testcase classname="OrderParamTest" name="placeOrderDifferentColorsPositiveTest[2] on placeOrderDifferentColorsPositiveTest[2](OrderParamTest)" time="1.921"/>
  <testcase classname="OrderParamTest" name="placeOrderDifferentColorsPositiveTest[3] on placeOrderDifferentColorsPositiveTest[3](OrderParamTest)" time="2.579"/>
</testsuite> <!-- Surefire test -->
